Composer/packages/lib/ui-shared/src/styled/styledComponents.tsx

```diff
@@ -41,7 +41,7 @@ export const BorderedDiv = styled.div<DivProps>(
 export const SingleLineDiv = styled.div<DivProps>`
   ${StandardFontCSS};
   ${TruncatedCSS};
-  line-height: ${(height) => (height ? height + 'px' : undefined)};
+  line-height: ${(height) => (height && typeof height === 'number' ? height + 'px' : undefined)};
```

Comment:
the fix is to destructure height, that function signature is basically (props: DivProps) => { styles here }

DivProps has children + height and width according.

Reply:
When debugging sometimes the height comes in as a number and sometimes it comes in as an object that doesn't have any height within it.

Reply:
I dont think it can be number:
https://emotion.sh/docs/styled